man/tpm2_startauthsession.1.md

```diff
@@ -0,0 +1,71 @@
+% tpm2_startauthsession(1) tpm2-tools | General Commands Manual
+%
+% JANUARY 2018
+
+# NAME
+
+**tpm2_startauthsession**(1) - Start a session with the TPM.
+
+# SYNOPSIS
+
+**tpm2_startauthsession** [*OPTIONS*]
+
+# DESCRIPTION
+
+**tpm2_startauthsession**(1) Starts a session with the TPM. The default is
+to start a *trial* session unless the **-a** option is specified. It outputs
+the session handle in a yaml format to stdout as key value "session-handle"
+and the handle in hex (0x1234) format.
```

Comment:
Because I don't know if tpm2_startauthsession will ever need to output something else. So I can add to a YAML output and keep backwards compatibility. In this case it's literally session-handle: 0x1234 which is super easy to parse without a full blown YAML parser. In fact the test scripts just use cut.

Reply:
Yeah, it eventually needs tpmNonce per [table 16](https://trustedcomputinggroup.org/wp-content/uploads/TPM-Rev-2.0-Part-3-Commands-01.38.pdf)